### PR TITLE
Addition of true-peak conformance tests to the test suite

### PR DESCRIPTION
**Pull request by audionuma**
True-peak tests are only run if the libebur128 has been built with speexdsp.
To be used with [ebu loudness test set v4 ](https://tech.ebu.ch/publications/ebu_loudness_test_set)

### CODE REVIEW

File: test/tests.c [hunk 1]
```diff
@@ -126,8 +128,11 @@ double test_true_peak(const char* filename) {
     ebur128_add_frames_double(st, buffer, (size_t) nr_frames_read);
   }
 
-  ebur128_loudness_range(st, &true_peak);
-
+  for (int i = 0; i < file_info.channels; i++) {
```
Comment (bmatherly) [inline test/tests.c]: 'for' loop initial declarations are only allowed in c99 mode. Consider declaring "i" at the top of the function as is done in the rest of the code.

Comment (audionuma) [inline test/tests.c]: Done. Is it better now ?

Comment (bmatherly) [inline test/tests.c]: Yes. Looks good. Thanks.

**Comment by jiixyj** [comment]
Thank you!

**Pull request merged**